packages/twenty-server/src/modules/person/person-domain-validation-manager/services/person-domain-validation.service.ts

```diff
@@ -0,0 +1,137 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+
+import {
+  CreateManyResolverArgs,
+  CreateOneResolverArgs,
+} from 'src/engine/api/graphql/workspace-resolver-builder/interfaces/workspace-resolvers-builder.interface';
+
+import { InjectObjectMetadataRepository } from 'src/engine/object-metadata-repository/object-metadata-repository.decorator';
+import { WorkspaceMemberRepository } from 'src/modules/workspace-member/repositories/workspace-member.repository';
+import { WorkspaceMemberWorkspaceEntity } from 'src/modules/workspace-member/standard-objects/workspace-member.workspace-entity';
+
+export interface PersonData {
+  emails?: {
+    primaryEmail?: string;
+    additionalEmails?: object | null;
+  };
+}
+
+@Injectable()
+export class PersonDomainValidationService {
+  constructor(
+    @InjectObjectMetadataRepository(WorkspaceMemberWorkspaceEntity)
+    private readonly workspaceMemberRepository: WorkspaceMemberRepository,
+  ) {}
+
+  /**
+   * Normalizes additionalEmails to an array of strings
+   * Handles: null, undefined, string, array, JSON string, or object
+   */
+  private normalizeEmails(emails: unknown): string[] {
```

Comment:
i dont like this unknown.

can't we add here what it really is: a string, like '['test@test.com','test2@test2.com']'

also add a comment how it works. resp. add a comment how the input looks like

then we JSON.parse the string into an array of strings


```suggestion
  private normalizeEmails(emails?: string | null): string[] {
```